Source/WebCore/platform/graphics/gstreamer/mse/GStreamerMediaDescription.cpp

```diff
@@ -74,15 +74,18 @@ AtomString GStreamerMediaDescription::extractCodecName()
     }
 
     GUniquePtr<gchar> description(gst_pb_utils_get_codec_description(originalCaps.get()));
-    auto codecName = String::fromLatin1(description.get());
+    auto codecName = AtomString::fromLatin1(description.get());
 
     // Report "H.264 (Main Profile)" and "H.264 (High Profile)" just as "H.264" to allow changes between both variants
     // go unnoticed to the SourceBuffer layer.
-    if (codecName.startsWith("H.264")) {
-        size_t braceStart = codecName.find(" (");
-        size_t braceEnd = codecName.find(")");
-        if (braceStart != notFound && braceEnd != notFound)
-            codecName.remove(braceStart, braceEnd - braceStart);
+    if (codecName.startsWith("H.264"_s)) {
+        size_t braceStart = codecName.find(" ("_s);
+        size_t braceEnd = codecName.find(')');
+        if (braceStart != notFound && braceEnd != notFound) {
+            String updatedCodecName = codecName;
+            updatedCodecName.remove(braceStart, braceEnd - braceStart);
```

Comment:
Need to update this to not use `remove`. Should use `makeAtomString`.

Reply:
I guess we want a makeAtomStringByRemoving() which returns an AtomString.